* don't rebuild flavors when installing and checking, and check using MPI
author: Miguel Dias Costa (National University of Singapore)
--- BerkeleyGW-4.0/Makefile.orig	2024-03-20 11:08:04.996274767 +0800
+++ BerkeleyGW-4.0/Makefile	2024-03-20 11:08:22.466174576 +0800
@@ -130,7 +130,7 @@
 	cp flavor_cplx.mk flavor.mk
 	$(MAKE) all
 
-install: all
+install:
 ifdef INSTDIR
 	mkdir -p $(INSTDIR)/bin
 	install bin/*.x $(INSTDIR)/bin/
@@ -152,8 +152,8 @@
 	$(error Error: Please define installation prefix INSTDIR via 'make install INSTDIR='.)
 endif
 
-check: all
-	cd testsuite && $(MAKE) check
+check:
+	cd testsuite && $(MAKE) check-parallel
 
 check-save: all
 	cd testsuite && $(MAKE) check-save
